ibis/impala/kudu_support.py

```diff
@@ -67,13 +68,16 @@ def connect(self, host_or_hosts, port_or_ports=7051, rpc_timeout=None):
           If you pass multiple host names, pass multiple ports
         rpc_timeout : kudu.TimeDelta
           See Kudu client documentation for details
+        admin_timeout_ms : kudu.TimeDelta
+          See Kudu client documentation for details
 
         Returns
         -------
         None
         """
         self.client = kudu.connect(host_or_hosts, port_or_ports,
-                                   rpc_timeout=rpc_timeout)
+                                   rpc_timeout_ms=rpc_timeout,
+                                   admin_timeout_ms=admin_timeout_ms)
```

Comment:
Sorry I should've been more clear in my initial comment. Can you change `admin_timeout_ms` to `admin_timeout` as well? The `ms` suffix doesn't really make sense to me since you have to pass in an object that has units already, plus it conflicts with the naming convention of `rpc_timeout`.

Once you make this change I'll merge in your fix.